src/gui/mzroll/pollyintegration.cpp

```diff
@@ -0,0 +1,263 @@
+#include "pollyintegration.h"
+
+PollyIntegration::PollyIntegration(TableDockWidget* tableDockWidget)
+{
+
+    _tableDockWidget = tableDockWidget;
+}
+
+PollyIntegration::~PollyIntegration()
+{
+    qDebug()<<"exiting PollyIntegration now....";
+}
+
+QByteArray PollyIntegration::run_qt_process(QString command){
+    QProcess process;
+    process.start(command);
+    process.waitForFinished();
+    QByteArray result = process.readAllStandardOutput();
+    QByteArray result2 = process.readAllStandardError();
+    return result;
+}
+
+QByteArray PollyIntegration::run_system_process(QString command){
+    QByteArray ba = command.toLatin1();
+    const char *c_str2 = ba.data();
+    system(c_str2);
+    return "";
```

Comment:
Is there any reason to return **""** always, we can simply ignore return statement?

Reply:
All these "system" functions are not being used as of now, We are using Qprocess.
But in some cases where QProcess fails, We may use these system functions ... This is a work in progress so I just returned empty string as of now..